lib/camerata.rb

```diff
@@ -158,10 +159,30 @@ def respond_to_missing?(method_name, include_private = false)
     end
 
     private
+    def check_for_special_compose(meth)
```

Comment:
Ah so specifying the solr and db compose for the respective deploy commands before merging and running commands.